testdata/book/multiple_include_a.yml

```diff
@@ -0,0 +1,18 @@
+desc: For multiple include test
+vars:
+  foo: 123
+  bar: "123-abc"
+  baz: "-23"
+  qux: 4
+  quux: "2"
```

Comment:
How about adding `if: included` to multiple_include_a.yml and multiple_include_b.yml so we can better understand the behavior of override vars?